packages/vouching/contracts/Vouching.sol

```diff
@@ -0,0 +1,101 @@
+pragma solidity ^0.4.24;
+
+
+import "./ZepToken.sol";
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+
+contract Vouching {
+  using SafeMath for uint256;
+
+  struct Dependency {
+    address owner;
+    address dependencyAddress;
+    uint256 stake;
+  }
+
+  mapping (string => Dependency) private _registry;
+  uint256 private _minimumStake;
+  ZepToken private _token;
+
+  modifier onlyDependencyOwner(string name) {
+    require(msg.sender == _registry[name].owner);
+    _;
+  }
+
+  event OwnershipTransferred(address indexed oldOwner, address indexed newOwner);
+  event DependencyCreated(string name, address indexed owner, address indexed dependencyAddress, uint256 initialStake);
+  event Vouched(string name, uint256 amount);
+  event Unvouched(string name, uint256 amount);
+  event DependencyRemoved(string name);
+
+  constructor(uint256 minimumStake, ZepToken token) public {
+    require(token != address(0));
+    _minimumStake = minimumStake;
+    _token = token;
+  }
+
+  function minimumStake() public view returns(uint256) {
+    return _minimumStake;
+  }
+
+  function token() public view returns(ZepToken) {
+    return _token;
+  }
+
+  function getDependencyAddress(string name) public view returns(address) {
+    return _registry[name].dependencyAddress;
+  }
+
+  function getDependencyOwner(string name) public view returns(address) {
+    return _registry[name].owner;
+  }
+
+  function getDependencyStake(string name) public view returns(uint256) {
+    return _registry[name].stake;
+  }
```

Comment:
How about replacing these 3 functions with a `getDependency` that returns a tuple `(address, address, uint256)`?